Stacks.js nav changes (#171)
hirosystems/docs: docs/stacks.js/getting-started.md

---
title: Getting Started
---

> You can get started with [Stacks.js starters](https://github.com/hirosystems/stacks.js-starters) to help bootstrap frontend applications.

Reviewer: It'd be nice to have a quick explanation of what the starter site can help with, and also replace the URL to https://docs.hiro.so/stacksjs-starters.

---
title: Get Started
---

> Stacks.js starters provides a way to start a new JavaScript project using the most common frameworks.

Reviewer: It'd be better make it Stacks.js specific.

```suggestion
[Stacks.js starters](https://docs.hiro.so/stacksjs-starters) provide a quick and easy way to get started with Stacks.js.
```

> The [Stacks.js starters](https://docs.hiro.so/stacksjs-starters) templates help you bootstrap your frontend application.

Reviewer: Perhaps remove this.
